blueprints/cloud-operations/network-dashboard/variables.tf

```diff
@@ -29,18 +29,21 @@ variable "monitoring_project_id" {
 
 variable "prefix" {
   description = "Customer name to use as prefix for monitoring project"
-  default     = ""
```

Comment:
The prefix is used to name the monitoring project. If we leave it to "", then the monitoring project creation will fail as another project with the same ID exists and the user will have to set the prefix anyway. I prefer to let it empty so that the user has to chose a prefix when running the terraform code. What do you think?

Reply:
I disagree as the prefix for creating the project is optional when `monitoring_project_id` is set no project will be created. So the user has to set a variable without purpose

Reply:
Ah, I got it now, in that case it makes sense.